WalletConnectSharp.NEthereum/Client/WalletConnectClient.cs

```diff
@@ -18,7 +19,7 @@ public WalletConnectClient(WalletConnectSession provider)
 
         protected override async Task<RpcResponseMessage> SendAsync(RpcRequestMessage message, string route = null)
         {
-            _id += 1;
+            _id = RpcPayloadId.Generate();
```

Comment:
Since we are generating a random id each time, we should convert this variable to a local variable. No need to store it as a class variable anymore